commands/prune.go

```diff
@@ -176,6 +176,10 @@ func toBuildkitPruneInfo(f filters.Args) (*client.PruneInfo, error) {
 
 	bkFilter := make([]string, 0, f.Len())
 	for _, field := range f.Keys() {
+		if field == filterKey {
```

Comment:
This is quite confusing to read. We should rename so that this reads `filter == untilKey`

Reply:
The current one LGTM, but noticed that I messed up and actually meant to write `field == untilKey` here that would have made a simpler patch. No need to change anything with current PR.